symplyphysics/laws/thermodynamics/volume_of_heated_body_from_initial_volume_and_heating_temperature.py

```diff
@@ -27,7 +36,20 @@
 law = Eq(final_volume,
     start_volume * (1 + expansion_coefficient * (final_temperature - start_temperature)))
 
-# TODO: derive from [definition of expansion coefficient](../../definitions/volumetric_coefficient_of_thermal_expansion.py)
+# Derive from [definition of expansion coefficient](../../definitions/volumetric_coefficient_of_thermal_expansion.py)
+
+_volume = coef_def.volume
+_temperature = coef_def.temperature
+
+_eqn = coef_def.definition.subs(coef_def.volumetric_expansion_coefficient, expansion_coefficient)
+
+_volume_expr = dsolve(_eqn, _volume(_temperature), ics={_volume(0): start_volume}).rhs
+
+_volume_series = _volume_expr.series(_temperature, 0, 2).removeO()
+
+_volume_derived = _volume_series.subs(_temperature, final_temperature - start_temperature)

```

Comment:
Why do we substitute with temperature difference? It should be final temperature IMO.

Reply:
Hm, I actually found a way to make this look sensible, will refactor it later.